### PR TITLE
Add Clerk.dev example

### PR DESCRIPTION
**Pull request by SokratisVidros**
## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Documentation added
- [ ] Telemetry added. In case of a feature if it's used or not.

## Documentation / Examples

- [x] Make sure the linting passes

### CODE REVIEW

File: examples/with-clerk/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+# Clerk.dev Example
+
+Add sign up, sign in, and profile management to your Next.jsapplication in minutes with [Clerk.dev](https://www.clerk.dev/?utm_source=github&utm_medium=nextjs-examples&utm_campaign=nextjs).
+
+## Getting Started
+
+Learn to install and initialize Clerk in a new Next.js application by reading our [official guide](https://docs.clerk.dev/frontend/quick-starts/next.js).
+
+## Learn More
```
Comment (leerob) [inline examples/with-clerk/README.md:10]: Please follow the format of the other examples in the README 😄

Comment (SokratisVidros) [reply examples/with-clerk/README.md:9]: Done.

Comment (SokratisVidros) [reply examples/with-clerk/README.md:21]: Oops my bad. Done.

**Review by leerob — APPROVED** [review]

**Comment by SokratisVidros** [comment]
Thanks @leerob

**Pull request merged**